### DISCUSSION TITLE
Questions about Copilot + Open Source Software Hierarchy

### DISCUSSION THREAD

**Discussion by liaochris** [msiemens/tinydb#568] [General]
Hi! My name is Chris and I'm writing a thesis on Open Source Software. I'm trying to collect/validate my data and I have two questions for the maintainers of this project.
1) Did this project receive free github copilot access on June 2022?
2) My thesis is especially focused on understanding hierarchical structures. Would it be possible to share a list of individuals in this project with triage/write/maintain/admin access in this project? I understand this may be confidential information, so please feel free to share it with [chrisliao@uchicago.edu ](chrisliao@uchicago.edu)

Happy to chat further if you have questions, and thank you for your time!

**Comment by msiemens** [comment]
@liaochris, personaly, I haven't used GitHub Copilot and I'm not aware of any contributors using it either. Re 2): As of right now, I'm the sole maintainer of this project.
